content/js/record.md

```diff
@@ -1,335 +1,189 @@
-<a name="overview"></a>
-## Overview
-
-Please make sure you know about and have already configured your skygear
-[container](/js/guide#set-up-app) before you proceed.
-
-<a name="record"></a>
-### The Record Class
-
-- `Record` must have a type.
-- Each `Record` object is like a dictionary with keys and values; keys will be
-mapped to database column names, and values will be stored appropriately
-based on the data type. Please refer to [Data Type](#data-type)
-section within this guide for more information.
-- `Record` will be owned by the currently logged in user.
-- Each `Record` object has its unique `id` (combination of record type
-  and uuid used in the database as `_id`).
-- `Record` has reserved keys that cannot be used, such as `id` and `_id`.
-Please refer to [Reserved Columns](#reserved) section for more.
-
-You can design different `Record` type to model your app. Just like defining
-tables in SQL.
-
-``` javascript
-const Note = skygear.Record.extend('note');
-const Blog = skygear.Record.extend('blog');
-
-const note = new Note({ 'content': 'Hello World' });
+<a name="what-are-records"></a>
+## What are Records?
+In Skygear, you can store data using **Records**. They provide a reliable way store *structured*, *persistent* data on the backend.
+
+Each Skygear record is associated with a named **type** which is bound to an extendable data schema. In the JS SDK, records are simply JS objects.
+
+All Skygear records are also assigned a unique UUID during creation for identification.
+
+<a name="record-owner"></a>
+### Record Owner
+By design, each Skygear record is associated a user for easy access control so you must be logged-in to be able to create and save records. However, if access control is not required, you can login as the anonymous user. 
```

Comment:
- Record owner is not mentioned in this paragraph, need to mention that the "user" is the record owner, and that the record owner can read/edit the record
- Need a link to Access Control page